css/index.css

```diff
@@ -188,44 +286,191 @@ header {
+    display: flex;
+    justify-content: center;
+}
+
+.mailing-list input{
+    width:70%;
+    font-size: 1.8rem;
+}
+
+.mailing-list button{
+    width:30%;
+    font-size: 1.8rem;
+}
+
+footer nav .btm.btns{
+    width: 60%;
+    display: flex;
+    justify-content: space-around;
+}
+
+footer nav .btm.btns a{
+    font-size: 1.6rem;
+}
+
+.copyright{
+    background-color: rgb(0, 4, 17);
+    color: rgb(225, 239, 230);
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    padding: 0 8%;
+}
+
+.copyright p{
+    font-size: 1.2rem;
 }
 
 /* mobile media query */
 @media (max-width: 428px) {
     /* header section */
+    header {
+            height: 105vh;
+        }
+    
+        header div {
+            height: 50vh;
+            display: flex;
+            justify-content: center;
+            flex-wrap: wrap;
+            align-items: flex-start;
+        }
+    
+        header div h1 {
+            width: 80%;
+            padding-top: 2%;
+        }
     
+        header div h1 a {
+            display: flex;
+            justify-content: center;
+            align-items: flex-start;
+            flex-wrap: wrap;
+            font-size: 3.5rem;
+        }
+    
+        nav {
+            width: 80%;
+            display: flex;
+            justify-content: center;
+            padding: 10% 0;
+        }
+    
+        .top.btns {
+            height: 20vh;
+        }
+    
+        .top.btns a {
+            height: 5vh;
+            width: 100%;
+            display: flex;
+            justify-content: center;
+            font-size: 2.2rem;
+        }
+    
+        nav div {
+            width: 50%;
+            height: 8vh;
+        }
+    
+        .social {
+            justify-content: space-evenly;
+            width: 80%;
+        }
+    
+        .social a i {
+            font-size: 2rem;
+        }
+    
+        /* Spacer */
+        .spacer {
+            height: 17vh;
+        }
 
     /* gallery section */
     .gallery {
         height: auto;
+        margin-top:4%;
+        background-color: rgb(0, 4, 17);
+    }
+    .gallery .foodTitle{
+        padding: 5%;
+        background-color: rgb(0, 4, 17);
+    }
+    .gallery .img{
+        margin:4%;
     }
     
     /* about us section */
 
     .about {
-        height: 200vh;
+        height: auto;
     }
-    
-    .about h2 {
+
+    .about .about-container h2 {
         width: 100%;
-        padding-top: 5vh;
+        padding: 15vh 0;
+        justify-content: flex-start;
     }
     
-    .about-container .text {
-        width: 85%;
+    .about .about-container .text{
+        width: 95%;
+    }
+
+    .text h3{
+        font-size:4.3rem;
+        padding-top:5vh;
+    }
+    .text p{
+        font-size:2.5rem;
     }
     
     /* contact us section */
     .contact h3 {
         padding-top: 2.5vh;
+        font-size: 4.5rem;
     }
-    
+
+    .contact p, .contacts a{
+        font-size:2.5rem;
+        padding: .4vh 0;
+    }
+
```

Comment:
When looking at the page on 428px width the contacts number and email are side by side instead of on top of each other. To fix that you could either put a class name in the div containing the a tags in order to grab the area or do the following: 

`.contacts div {
        display: flex;
        flex-wrap: wrap;
    }
    
    .contacts div a {
        width: 100%;
    }`
    
    
Adding the flex and wrap to the div allows them to stack and you need to set the a tag width to 100% for them to actually stack.